packages/next/next-server/lib/router/router.ts

```diff
@@ -674,8 +678,9 @@ export default class Router implements BaseRouter {
         return
       }
 
-      const route = toRoute(pathname)
-      this.pageLoader.prefetch(route).then(resolve, reject)
+      this.pageLoader[options.priority ? 'loadPage' : 'prefetch'](
```

Comment:
```suggestion
      this.pageLoader[options.urgent ? 'loadPage' : 'prefetch'](
```
Should we call this `urgent` to stress we're loading it as fast as we can?

Reply:
Per OP I used `priority` to mimic quicklink naming:
https://github.com/GoogleChromeLabs/quicklink/blob/836f170df4007c05c24b1300e0faf04fb823168b/src/index.mjs#L47

Not sure I'm a fan of `urgent`, sounds like that means the navigation _is_ happening haha.